### PR TITLE
Fix comments between headers (#447)

### PR DESCRIPTION
**Pull request by mbodock**
#### Background

Closes issue #447

Sometimes is nice to provide more information about a specific header, specially when sharing targets files. For this reason would be nice be able to put comments in the header section.

#### Checklist

- [x] Git commit messages conform to [community standards](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).
- [x] Each Git commit represents meaningful milestones or atomic units of work.
- [x] Changed or added code is covered by appropriate tests.

### CODE REVIEW

File: lib/targets.go [hunk 1]
```diff
@@ -300,6 +300,8 @@ func NewHTTPTargeter(src io.Reader, body []byte, hdr http.Header) Targeter {
 		for sc.Scan() {
 			if line = strings.TrimSpace(sc.Text()); line == "" {
 				break
+			} else if strings.HasPrefix(strings.TrimSpace(sc.Text()), "#") {
```
Comment (tsenart) [inline lib/targets.go]:
```suggestion
			} else if strings.HasPrefix(line, "#") {
```

**Comment by mbodock** [comment]
@tsenart Applied the suggestion, nice catch!

Also rebased with master in upstream. Please let me know if you need anything else.

**Comment by tsenart** [comment]
One last request: Please update the relevant sections of the README with this added functionality.

**Comment by mbodock** [comment]
Just removed the part about targets.

This should be enough since  my first usage I already expected to be able to use comments anywhere.

**Comment by mbodock** [comment]
Hello @tsenart, thank you very much for your time.

**Pull request merged**